cli/flags.rs

```diff
@@ -848,13 +869,6 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
         .help("Stop on first error")
         .takes_value(false),
     )
-    .arg(
-      Arg::with_name("quiet")
```

Comment:
I feel the functionality between these 2 flags is not exactly the same... I wonder if you should give one of them a different name.

Reply:
I wondered about that. AFAICT, the `--quiet` flag on the test command is currently unused, so its intended purpose was a bit unclear.

Reply:
Quiet used to be passed to python unit tests where it would mean each test prints one char ( . E or S ), rather than a line printed for each test.

Reply:
This means removing the existing flag should be safe, right?